src/pipecat/utils/asyncio/task_manager.py

```diff
@@ -285,7 +287,8 @@ async def wait_for_task(self, task: asyncio.Task, timeout: Optional[float] = Non
         name = task.get_name()
         try:
             if timeout:
-                await asyncio.wait_for(task, timeout=timeout)
+                async with pipecat.utils.asyncio.timeout.timeout(timeout):
+                    await task
```

Comment:
Same here.

Reply:
Yep, previous commit that I forgot to update.